app/components/polls/polls-list/polls-list.less

```diff
@@ -0,0 +1,5 @@
+polls-list {
+  button.btn-danger {
+    display: none;
```

Comment:
I don't think so... but I removed that line. We'll probably want to remove polls and the api supports deleting polls anyway.